documentation/help-documenting.rst

```diff
@@ -84,4 +84,12 @@ a pull request directly. It's best to avoid filing a single issue for an entire
 section containing multiple problems; instead, file several issues so that it
 is easier to break the work up for multiple people and more efficient review.
 
+For help with the finer points of English technical writing, mention the
+`@python/proofreaders <https://github.com/orgs/python/teams/proofreaders>`__
+team in your issue or pull request in any `@python <https://github.com/python/>`__
+repo. If you'd like to join the team,
+`open a core-workflow issue <https://github.com/python/core-workflow/issues/new/choose>`__
+similar to
+`python/core-workflow#461 <https://github.com/python/core-workflow/issues/461>`__.
+
 .. _issue tracker: https://github.com/python/cpython/issues
```

Comment:
```suggestion
For help with the finer points of English technical writing, mention the
`@python/proofreaders`_ team in your issue or pull request in any `@python`_ repo.
If you'd like to join the team, `open a core-workflow issue <proofreaders-issue>`_.

.. _issue tracker: https://github.com/python/cpython/issues
.. _@python/proofreaders: https://github.com/orgs/python/teams/proofreaders
.. _@python: https://github.com/python/
.. _proofreaders-issue: https://github.com/python/core-workflow/issues/new?assignees=&labels=&projects=&template=proofreader_membership.md&title=Request+for+Python+Proofreaders+team+membership%3A+name
```

I updated the link to the new template and moved it at the end together with the others.  I hope I got the syntax right :upside_down_face: 

I was also wondering if there are any restrictions for joining the group -- do people that send requests need to be already part of the `python` org (e.g. as triagers)? If so, we should mention that too.

Reply:
Good question, asked at https://github.com/python/devguide/issues/886#issuecomment-1820331651

Reply:
Answer:

> At the moment, there aren't any restrictions. (Of course, that might change if people start abusing this somehow.)

https://github.com/python/devguide/issues/886#issuecomment-1874954336

So nothing to mention right now.